app/views/hosts/_list.html.erb

```diff
@@ -31,8 +33,16 @@
         </td>
         <td class="hidden-xs ellipsis"><%= (icon(host.operatingsystem, :size => "16x16") + " #{host.operatingsystem.to_label}").html_safe if host.operatingsystem %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= host.compute_resource_or_model %></td>
+        <td class="hidden-tablet hidden-xs ellipsis">
+          <% if host.owner_type == 'User' %>
+            <%= icon_text('user', host.owner, :kind => 'fa') %>
+          <% else %>
+            <%= icon_text('users', host.owner, :kind => 'fa') %>
+          <% end %>
+        </td>
         <td class="hidden-tablet hidden-xs"><%= label_with_link host.hostgroup, 23, @hostgroup_authorizer %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= last_report_column(host) %></td>
+        <td class="hidden-tablet hidden-xs" title="<%= host.comment.truncate(255) %>"><%= icon_text('comment', '') unless host.comment.empty? %></td>
```

Comment:
```suggestion
        <td class="hidden-tablet hidden-xs" title="<%= host.comment&.truncate(255) %>"><%= icon_text('comment', '') unless host.comment.empty? %></td>
```
You can use safe navigation in Ruby, so in case there is no comment for the host, it won't raise an exception.

Reply:
Thanks @ofedoren for suggestion. I am not sure how to fix that autofading, because in my environment seems like that it does not fade. So when I hover over the icon, the tooltip stays there until I move away my cursor.